rc4me/fetch_rcs.py

```diff
@@ -0,0 +1,88 @@
+"""Clone dotfile repo to rc4me home directory."""
+
+from pathlib import Path
+import logging
+import click
+import git
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.argument("repo", type=str)
+def _fetch_rcs_cli(rc4me_home: Path, repo: str) -> None:
```

Comment:
Don't you need an option/argument for `rc4me_home`?

Reply:
Fixed.